kibana-reports/public/components/app.tsx

```diff
@@ -47,10 +47,10 @@ export interface CoreInterface {
 
 interface OpendistroKibanaReportsAppDeps {
   basename: string;
-  notifications: CoreStart['notifications'];
   http: CoreStart['http'];
-  navigation: NavigationPublicPluginStart;
+  // navigation: NavigationPublicPluginStart;
```

Comment:
If navigation and notifications are not used anywhere, please remove them from code, instead of comment it out.
@joshuali925 @davidcui-amzn could you confirm that it's safe to remove those 2?

Reply:
@davidcui-amzn can you take a look, and if it's safe to remove this, let's merge in the PR first

Reply:
Can confirm, `navigation` and `notifications` are not used